src/modules/SX126x/SX126x.h

```diff
@@ -806,6 +806,13 @@ class SX126x: public PhysicalLayer {
    */
    uint32_t getTimeOnAir(size_t len);
 
+   /*!
+     \brief Get instantaneous RSSI value during recption of the packet. Should switch to FSK receive mode for LBT implementation.
+
+     \returns Instantaneous RSSI value in (-2)*dBm.
+   */
+   uint8_t getRSSIInst();
```

Comment:
This should be `float` too ;)

Reply:
Yeah, I got mails from Github saying some checks have failed and was wondering if editing directly to a PR was the right way to do it.
I should get more familiar with Github :)